domain/cloudimagemetadata/service/service.go

```diff
@@ -38,7 +38,6 @@ type Service struct {
 	st State
 }
 
-// NewService returns a new service around the persistence layer
```

Comment:
Add back this comment, or create a new one.

Reply:
My bad thanks to notice